### PR TITLE
npm script - replace prepublish

### LINKED ISSUE #242: npm prepublish script needs to be updated
**Is your feature request related to a problem? Please describe.**
When we run `npm install`, there is this warning:

```
npm WARN prepublish-on-install As of npm@5, `prepublish` scripts are deprecated.
npm WARN prepublish-on-install Use `prepare` for build steps and `prepublishOnly` for upload-only.
npm WARN prepublish-on-install See the deprecation note in `npm help scripts` for more information.
```

**Describe the solution you'd like**
Update the scripts to use `prepare`. 

**Testing notes**
[Tester to complete]

Dev insight: When we publish a package, the prepare step should run before publishing.

**Additional context**

- [x] Initially labelled with ["Refinement needed"](https://github.com/BBC-News/psammead/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by sareh**
Resolves #242

- Replaces deprecated prepublish script with prepare script

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,9 +10,9 @@
     "deploy-storybook": "build-storybook -o storybook_dist && gh-pages -d storybook_dist",
     "force-package-lock-https": "find . -type f -name \"package-lock.json\" | grep -v node_modules | xargs -I % sh -c 'echo \"Replacing http->https in %\"; sed -i \"\" -e \"s/http:\\/\\//https:\\/\\//g\" %'",
     "install:packages": "lerna exec -- npm install",
-    "postinstall": "npm run ci:packages",
+    "postinstall": "npm run ci:packages && npm run build",
```
Comment (dr3) [inline package.json]: Isnt `npm run build` already a postinstall step?

Comment (dr3) [reply package.json]: image
This causes the build to run twice

**Comment by radiocontrolled** [comment]
@sareh I can confirm that your pr removes 

```
npm WARN prepublish-on-install As of npm@5, `prepublish` scripts are deprecated.
npm WARN prepublish-on-install Use `prepare` for build steps and `prepublishOnly` for upload-only.
npm WARN prepublish-on-install See the deprecation note in `npm help scripts` for more information.
```
and also that it builds twice, per @dr3's observation

**Comment by sareh** [comment]
@dr3 @radiocontrolled Thanks for your reviews. I've made the change.

**Review by bcmn — APPROVED** [review]

**Review by ChrisBAshton — APPROVED** [review]

**Comment by jamesbrumpton** [comment]
LGTM 👍

**Pull request merged**